code/game/gamemodes/wizard/godhand.dm

```diff
@@ -77,6 +77,45 @@
 	new /obj/structure/closet/statue(L.loc, L)
 	..()
 
+/obj/item/melee/touch_attack/plushify
+	name = "fabric touch"
+	desc = "The power to sew your foes into a doom cut from the fabric of fate."
+	catchphrase = "MAHR-XET 'ABL"
+	on_use_sound = 'sound/magic/smoke.ogg'
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	color = COLOR_PURPLE
+
+/obj/item/melee/touch_attack/plushify/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity || target == user || !isliving(target) || !iscarbon(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED)) //There are better ways to get a good nights sleep in a bed.
+		return
+	var/mob/living/L = target
+	L.Plushify()
+	..()
+
+
+/mob/living/proc/Plushify()
+	var/mob/living/simple_animal/shade/sword/bread/plushvictim = new(get_turf(src))
+	var/obj/item/toy/plushie/plush_type = pick(subtypesof(/obj/item/toy/plushie) - typesof(/obj/item/toy/plushie/fluff) - typesof(/obj/item/toy/plushie/carpplushie)) //exclude the base type.
+	if(issilicon(src))
+		plush_type = /obj/item/toy/plushie/borgplushie
+	else if(isalien(src))
+		plush_type = /obj/item/toy/plushie/face_hugger
+	else if(ishuman(src))
+		var/mob/living/carbon/human/H = src
+		plush_type = H.dna.species.plushie_type
```

Comment:
we dont allow `istype(src)` in our codebase, these are alias'es to that.
I suggest making an override variable that is passed from the parent proc
```
/mob/living/proc/Plushify(plushie_override)
	var/mob/living/simple_animal/shade/sword/bread/plushvictim = new(get_turf(src))
	var/obj/item/toy/plushie/plush_type = pick(subtypesof(/obj/item/toy/plushie) - typesof(/obj/item/toy/plushie/fluff) - typesof(/obj/item/toy/plushie/carpplushie))
	if(plushie_override)
		plush_type = plushie_override
```
and stuff like
```
/mob/living/carbon/human/Plushify(plushie_override)
	. = ..(H.dna.species.plushie_type)
```

Reply:
Done